packages/office-addin-usage-data/src/main.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export * from "./defaults";
+export * from "./officeAddinUsage-Data";
```

Comment:
we need to add export for usageDataJsonData as well

Reply:
Let's rename that file "usage-data.ts"

Reply:
Actually, "usageData.ts" is better.

Reply:
renaming